### ISSUE TITLE
Batch normalization should behave differently in training and testing phase.

### ISSUE THREAD

**Issue by SchafferZhang** [shaohua0116/SSGAN-Tensorflow#11]
I notice that you involve batch normalization in both discriminator and generator. As we all know, batch normalization should behave differently in training phase and testing phase. But according to your implementation, I don't see the differences.
Looking forward to your reply.

**Comment by patrafter1999** [comment]
Model() has an argument, 'is_train', which gets passed to batch_norm() to control training/test. Currently is_train is set to True by default in evaler.py when it creates a Model(). I tried is_train=False in evaler.py, which I found results in a slightly better test accuracy. So I guess the original author didn't put correct argument for evaluation.

**Comment by shaohua0116** [comment]
During testing, we do not update the moving average and moving variance of batch norm layers.
